{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-07T12:15:00.000+0000",
      "latitude": 51.44988503889691,
      "estimatedRestoration": "2022-04-10T21:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR3804'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR3804",
      "reference": "JR3804",
      "name": "RG18 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG18 0UD",
        "RG18 0UY",
        "RG18 0XU",
        "RG18 0XX",
        "RG18 0XY",
        "RG18 9NW",
        "RG18 9UA",
        "RG18 9UB",
        "RG18 9UD",
        "RG18 9UE",
        "RG18 9UF",
        "RG18 9UG",
        "RG18 9UH",
        "RG18 9UJ",
        "RG18 9UL",
        "RG18 9UN",
        "RG18 9UP",
        "RG18 9UR",
        "RG18 9UT",
        "RG18 9UU",
        "RG18 9UW",
        "RG18 9UX",
        "RG18 9UY",
        "RG18 9UZ",
        "RG18 9WX",
        "RG18 9XA",
        "RG18 9XB",
        "RG18 9XD",
        "RG18 9XE",
        "RG18 9XF",
        "RG18 9XH",
        "RG18 9XQ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.2299314900208123,
          51.44988503889691
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-08T08:00:00.000+0000",
      "UUID": "c9696894-af7c-40e0-92c4-33e6a9f74f66",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.2299314900208123,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-09T13:27:00.000+0000",
      "latitude": 51.414889170416714,
      "estimatedRestoration": "2022-04-09T22:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR4940'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR4940",
      "reference": "JR4940",
      "engineerOnSiteTime": "2022-04-09T06:15:00.000+0000",
      "name": "RG2 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG2 8PD",
        "RG2 8UB",
        "RG2 8UF"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.9658738696148785,
          51.414889170416714
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-09T06:30:00.000+0000",
      "UUID": "9b8a8bff-ee92-40ea-bccd-8efcbcc5dc30",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.9658738696148785,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-09T16:04:00.000+0000",
      "latitude": 51.75199466532249,
      "estimatedRestoration": "2022-04-10T03:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR5223'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR5223",
      "reference": "JR5223",
      "engineerOnSiteTime": "2022-04-09T17:55:00.000+0000",
      "name": "OX4 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "OX4 1BA",
        "OX4 1BE",
        "OX4 1BG",
        "OX4 1BH",
        "OX4 1BQ",
        "OX4 1BX"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.2389014448912992,
          51.75199466532249
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-09T17:45:00.000+0000",
      "UUID": "0f50717f-649a-41f4-af9c-f7e09835148f",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.2389014448912992,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-09T17:00:00.000+0000",
      "latitude": 51.41074573100139,
      "estimatedRestoration": "2022-04-09T22:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR5217'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR5217",
      "reference": "JR5217",
      "engineerOnSiteTime": "2022-04-09T17:00:00.000+0000",
      "name": "RG41 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG41 3AB",
        "RG41 3AD",
        "RG41 3AH",
        "RG41 3AW",
        "RG41 3BA"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.8640801541184719,
          51.41074573100139
        ],
        "type": "Point"
      },
      "UUID": "6500a26b-6c9f-4a42-b8cd-8a9c443bf930",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8640801541184719,
      "resolved": false
    }
  ]
}
